tea/scenes/new_profile.go

```diff
@@ -17,7 +17,7 @@ type newProfile struct {
 	title  string
 }
 
-func NewNewProfile(root components.RootModel, parent tea.Model) tea.Model {
+func NewProfile(root components.RootModel, parent tea.Model) tea.Model {
```

Comment:
Do not rename this, please follow the Go convention of calling struct constructors `New<struct>`